code/contracts/ONEWallet.sol

```diff
@@ -24,12 +25,12 @@ contract ONEWallet {
         bool completed;
     }
 
-    uint32 constant REVEAL_MAX_DELAY = 60;
+    uint32 constant REVEAL_MAX_DELAY = 60; 
 
     //    bool commitLocked; // not necessary at this time
     Commit[] commits; // self-clean on commit (auto delete commits that are beyond REVEAL_MAX_DELAY), so it's bounded by the number of commits an attacker can spam within REVEAL_MAX_DELAY time in the worst case, which is not too bad.
 
-    uint256[64] ______gap; // reserved space for future upgrade
+    uint256[64] ______gap; // reserved space for future upgrade // IH: I am not sure how this work
```

Comment:
This is following the practice in https://docs.openzeppelin.com/upgrades-plugins/1.x/writing-upgradeable and https://github.com/OpenZeppelin/openzeppelin-contracts-upgradeable/blob/v3.3.0/contracts/token/ERC20/ERC20Upgradeable.sol . Unimportant right now, since it is unlikely the wallet instance will be used in another smart contract any time soon. I added this field just to keep this use case in mind.

Reply:
Will remove soon, since we shouldn't need to worry about upgrade using contract at this time. We can do it later when users report this as an issue